Sensus.UITests/Tests.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using NUnit.Framework;
+using Xamarin.UITest;
+using Xamarin.UITest.Queries;
+
+namespace Sensus.UITests
+{
+    [TestFixture(Platform.Android)]
+    [TestFixture(Platform.iOS)]
+    public class SensusTests
+    {
+        IApp app;
+        Platform platform;
+
+        /// <summary>
+        /// this is our xamarin test cloud APIkey.
+        /// It could be found in the xamarin test cloud account.
+        /// Each team will have a different APIkey. The one below is from the sensus team.
+        /// </summary>
+        const string _APIkey = "b38984cea752e9d489573615fb5fddaf";
```

Comment:
You should check whether including this API key in the repository is a security risk.